Taarafo.Core.Tests.Unit/Services/Foundations/Profiles/ProfileServiceTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq.Expressions;
+using Microsoft.Data.SqlClient;
+using System.Runtime.Serialization;
+using Moq;
+using Taarafo.Core.Brokers.Loggings;
+using Taarafo.Core.Brokers.Storages;
+using Taarafo.Core.Models.Profiles;
+using Taarafo.Core.Services.Foundations.Profiles;
+using Tynamix.ObjectFiller;
+using Xeptions;
+using Xunit;
+using Taarafo.Core.Brokers.DateTimes;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Profiles
+{
+    public partial class ProfileServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly Mock<IDateTimeBroker> dateTimeBrokerMock;
+        private readonly IProfileService profileService;
+
+        public ProfileServiceTests()
+        {
+            this.storageBrokerMock = new Mock<IStorageBroker>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+            this.dateTimeBrokerMock = new Mock<IDateTimeBroker>();
+
+            this.profileService = new ProfileService(
+                storageBroker: this.storageBrokerMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object,
+                dateTimeBroker: this.dateTimeBrokerMock.Object);
+        }
```

Comment:
`dateTimeBroker` should always come before `loggingBroker` why?
Because `dateTimeBroker` while it's a utility broker it's fundamental to your business flow (validation). Therefore it's much more important to your business than the `loggingBroker`.

Reply:
done